CMakeLists.txt

```diff
@@ -6,13 +6,9 @@
 cmake_minimum_required (VERSION 3.0.0)
 
 # Set software information
-set(PROJECT_NAME libsndfile)
-SET(LIB_VERSION_MAJOR 1)
-SET(LIB_VERSION_MINOR 0)
-SET(LIB_VERSION_PATCH 28pre1)
-SET(LIB_VERSION ${LIB_VERSION_MAJOR}.${LIB_VERSION_MINOR}.${LIB_VERSION_PATCH})
-set(LIB_VERSION 1.0.28pre1)
-project(${PROJECT_NAME})
+project(libsndfile VERSION 1.0.28)
+SET(LIB_VERSION_PATCH pre1)
+set(LIB_VERSION ${PROJECT_VERSION}pre1)
```

Comment:
No, because PROJECT_VERSION format is X.X.X.X, where X is digit. LIB_VERSION_PATCH is still here if you want use it anywhere else.